plugins/woocommerce/src/StoreApi/Schemas/V1/AbstractAddressSchema.php

```diff
@@ -216,12 +223,27 @@ function( $key ) use ( $additional_fields_keys ) {
 
 		$schema = [];
 		foreach ( $address_fields as $key => $field ) {
-			$schema[ $key ] = [
+			$field_schema = [
 				'description' => $field['label'],
 				'type'        => 'string',
 				'context'     => [ 'view', 'edit' ],
 				'required'    => true,
 			];
+
+			if ( ! empty( $field['type'] ) && 'select' === $field['type'] ) {
+				$field_schema['type'] = 'string';
+				$field_schema['enum'] = array_map(
+					function( $option ) {
+						return $option['value'];
+					},
+					$field['options']
+				);
+				$field_schema['items'] = array(
+					'type' => 'string',
+				);
```

Comment:
Are you sure a select has items?

Reply:
Ah this is just for arrays I believe, I removed it